portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteApps.path

```diff
@@ -8,6 +8,21 @@
 <tr><td rowspan="1" colspan="3">RemoteApps</td></tr>
 </thead>
 <tbody>
+<tr>
+	<td>EMPTY_REMOTE_TABLE_MESSAGE</td>
+	<td>//div[contains(@class,'sheet-text') and contains(.,'No items were found.')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>REMOTE_TABLE_DELETE</td>
+	<td>//a[contains(@class,'dropdown-item') and contains(.,'Delete')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>REMOTE_TABLE_ELLIPSI</td>
+	<td>//button[contains(@class,'component-action')]</td>
+	<td></td>
+</tr>
```

Comment:
I think we have some low level general paths that can be used instead. Check out LexiconTable.path, MenuItem.path and/or Dropdown.path.